### PR TITLE
start wdi14 update

### PR DESCRIPTION
**Pull request by amaseda**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -23,35 +23,35 @@ $ git clone git@github.com:ga-wdi-exercises/whenpresident.git
...
+Then, start the Mongo server in a separate Terminal tab or window...
 
-Check out what the starter code looks like in the browser by running `$ nodemon` and then visiting `http://localhost:3001`.
+```bash
+$ mongod
+```
+
+Check out what the starter code looks like in the browser by running `$ nodemon` and then visiting `http://localhost:3001` in the browser.
 
 ## Express Review (25 minutes / 0:30)
```
Comment (nayana487) [inline readme.md]: The `Note` section needs to be updated. The Mongoose lesson had both promises and callbacks

Comment (nayana487) [inline readme.md]: Take out the "A" in MEAN under the MVC chart section? There's no Angular in the code they're looking at.

File: readme.md [hunk 2]
```diff
@@ -182,23 +185,25 @@ We can test this by...
 ### Questions
 
 <details>
+
   <summary><strong>Why are we able to write out `mongoose.model("Candidate")` in `seed.js`?</strong></summary>
```
Comment (amaseda) [inline readme.md]: Note to self: change `seed.js` to `seeds.js`

**Comment by amaseda** [comment]
Note to self: put "You Do" solutions in details-summary

**Comment by amaseda** [comment]
Note to self: Fix `How are '' and 'req.body' related?` details-summary

File: readme.md (hunk 1 above)
Comment (amaseda) [reply readme.md]: Parentheses are meant to indicate that it's not being applied here. Can still remove though.

**Pull request merged**